packages/amplify_auth_cognito/android/src/main/kotlin/com/amazonaws/amplify/amplify_auth_cognito/types/FlutterSignInRequest.kt

```diff
@@ -55,7 +55,7 @@ data class FlutterSignInRequest(val map: HashMap<String, *>) {
       try {
         var user: AuthUser? = Amplify.Auth.currentUser;
         if (user != null) {
-          throw FlutterInvalidStateException("There is already a user  signed in.", "Sign out before calling sign in.")
+          throw FlutterInvalidStateException("There is already a user signed in.", "Sign out before calling sign in.")
```

Comment:
🕵️  Is it possible that someone could be looking for the previous string with the extra space for some reason in their code?